modules/system/rolls/test-wfrp4e.js

```diff
@@ -256,6 +258,13 @@ export default class TestWFRP {
         }
       }
 
+      if (this.options.engagedModifier) {
+        let unmodifiedTarget = target - this.options.engagedModifier;
+        if (this.result.roll > unmodifiedTarget) {
+          this.result.other.push(game.i18n.localize("ROLL.HitAnotherEngagedTarget"))
+        }
+      }
```

Comment:
Shouldn't this be in the `WeaponTest` class?

Reply:
i kept it here, because of the sizemodifier being used same place and also to handle ranged trait.